### DISCUSSION TITLE
可否用Python程序打开插件并下载视频实现自动化呢

### DISCUSSION THREAD

**Discussion by hahaha0721** [xifangczy/cat-catch#588] [Announcements]
用gpt写了自动打开插件的代码，但插件总是无法正确打开:
from selenium import webdriver
from selenium.webdriver.chrome.options import Options

# 设置 Chrome 扩展路径
chrome_options = Options()
chrome_options.add_argument(r"C:\path\to\your\extension")  # 插件路径

driver = webdriver.Chrome(options=chrome_options)
driver.get("https://your-target-page.com")
修改后的代码基本也形似这样，无法正确打开插件

**Comment by xifangczy** [comment]
额 我惊了。。。😨
我未成想过的路。。

**Comment by qe201020335** [comment]
Python也许用[youtube-dl](https://github.com/ytdl-org/youtube-dl)或者[ytb-dlp](https://github.com/yt-dlp/yt-dlp)会更合适一些呢
